include/geode/io/image/private/vti_output_impl.h

```diff
@@ -43,7 +43,7 @@ namespace geode
             {
             }
 
-            void write_image_header( pugi::xml_node& piece,
+            pugi::xml_node& write_image_header( pugi::xml_node& piece,
                 const Point< dimension >& origin,
                 const std::array< index_t, dimension >& extent,
                 const std::array< double, dimension >& spacing )
```

Comment:
How this spacing is given? If you start to return the header xml node, maybe we need to review where direction and spacing should be set